web/src/user_profile.js

```diff
@@ -803,26 +803,30 @@ export function initialize() {
         e.preventDefault();
         const stream_id = Number.parseInt(user_profile_subscribe_widget.value(), 10);
         const sub = sub_store.get(stream_id);
+
+        if (!sub) {
+            return;
```

Comment:
Why do we need this now when it worked fine before? Or there is some bug which was present before and is fixed by this change.